workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/sales-enablement/fragments/search-results-page/index.css

```diff
@@ -0,0 +1,14 @@
+.search-page {
+	gap: var(--spacer-5);
+}
+
+.search-page-filters,
+.search-page-results {
```

Comment:
do we need to break line?

Reply:
Hello @nNilton ,

Breaking lines when we use different `cssClasses` for the same style is a good practice to improve readability and make the code friendlier for future maintenance. However, I understand that in this case the code might not be large enough to really require it. Since it’s more about readability than necessity, do you think it would be better to keep it in a single line?